lib/src/style/interactive_features/interactive_features.dart

```diff
@@ -0,0 +1,44 @@
+part of mapbox_maps_flutter;
```

Comment:
@pjleonard37 Is this the manually written or generated code? I think we need to take a similar approach to Android/iOS SDK and generate this.